JS-Algorithms/challenges.js

```diff
@@ -95,7 +106,15 @@ Since 10! === 3628800 and you sum 3 + 6 + 2 + 8 + 8 + 0 + 0
 ***** */
 
 const digitSum = (n) => {
-  // YOUR CODE HERE...
+   let factorial = n;
+   let arrayFactorial = [];
+   let sum = 0;
+   for (let i = factorial - 1; i >= 1; i--) {
+      factorial *= i;
+   }
```

Comment:
Prefer the use of `forEach` instead of a traditional `for`. And if you are dealing with `async/await` use `for of`

Reply:
Here I'm iterating and acumulating the factorial in that variable, I'm not iterating through an array